data/boundaries/citycountystate/index.html

```diff
@@ -16,8 +16,6 @@
   updates:
     - Update to Magna Metro Township October 2018
     - Updates to this dataset are published as needed.
-  opensgid:
-    table: boundaries.metro_townships
```

Comment:
**nitpick**: let's keep this property. I'd like to have this available on every dataset. Will you please add it back